src/Language/Haskell/HGrep/Query.hs

```diff
@@ -1,10 +1,13 @@
 {-# LANGUAGE NoImplicitPrelude #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
Is this being used? If you're just using it for interactive development, you could put it in your ghci.conf https://github.com/thumphries/dotfiles/blob/b3e7cbb7ca36e4c64c68e0409dc208ee3dd24a57/ghc/.ghc/ghci.conf

Reply:
no, i probably shouldn't have pushed this without cleaning it up a bit more. apologies, it was pretty late and i was tired.

that's a good tip about the ghci.conf tho, i didn't know about that! thanks